web/src/app/administration/components/workbasket-details/workbasket-details.component.spec.ts

```diff
@@ -37,16 +36,46 @@ import { NotificationService } from '../../../shared/services/notifications/noti
 export class DummyDetailComponent {
 }
 
+function createWorkbasket(workbasketId?, created?, key?, domain?, type?, modified?, name?, description?,
+  owner?, custom1?, custom2?, custom3?, custom4?, orgLevel1?, orgLevel2?, orgLevel3?, orgLevel4?,
+  _links?: Links, markedForDeletion?: boolean) {
+  const workbasket: Workbasket = {
+    workbasketId,
+    created,
+    key,
+    domain,
+    type,
+    modified,
+    name,
+    description,
+    owner,
+    custom1,
+    custom2,
+    custom3,
+    custom4,
+    orgLevel1,
+    orgLevel2,
+    orgLevel3,
+    orgLevel4,
+    markedForDeletion,
+    _links
+  };
+  return workbasket;
+}
+
 describe('WorkbasketDetailsComponent', () => {
   let component: WorkbasketDetailsComponent;
   let fixture: ComponentFixture<WorkbasketDetailsComponent>;
   let debugElement;
   let masterAndDetailService;
   let workbasketService;
   let router;
-  const workbasket = new Workbasket('1', '', '', '', ICONTYPES.TOPIC, '', '', '', '', '', '', '', '', '', '', '', '',
+  const workbasket = createWorkbasket('1', '', '', '', ICONTYPES.TOPIC, '', '', '', '', '', '', '', '', '', '', '', '',
```

Comment:
```suggestion
  const workbasket: Workbasket ={'1', '', '', '', ICONTYPES.TOPIC, '', '', '', '', '', '', '', '', '', '', '', '',{}};
```
why not directly provide the object?

Reply:
noted, will be changed